src/cm.css

```diff
@@ -8,6 +8,7 @@
   --text-color: #010101;
   --modal-padding: 21px;
   --content-padding: 38px;
+  --outlineStyle: 2px solid rgba(51, 157, 255, 0.7);
```

Comment:
please dont add any more global variables, this broken the last deploy. 

i made another pr to remove modal-padding and content-padding to prevent more breakage. 

right now the text color and primary color are getting pulled in from the admin dashboard variables. its likely possible to support generic variables like this, but the way that we built the parser for custom styles is not resilient enough to handle these right now.

for the time being, i would recommend avoid global variables, and if you want to add them back, we should open a ticket and determine how to design them into the admin dashboard like we do the text-color and primary-color